docs/irf/plot_fermi_psf.py

```diff
@@ -11,4 +11,4 @@
 # .to_image() computes the exposure weighted mean PSF
 kernel = psf.get_psf_kernel(geom=geom).to_image()
 
-kernel.psf_kernel_map.plot();
\ No newline at end of file
+kernel.psf_kernel_map.plot()
```

Comment:
Taking it strictly I think this is a false positive: the semicolon suppresses the output returned by the `.plot()` method. But I think we are nowhere uniform in Gammapy about this, so I'm fully OK to keep this for now.